recipes/ipypdf/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: ipypdf
+  version: {{ version }}
+
+source:
+  url: https://github.com/joelstansbury/ipypdf/archive/15e2826.tar.gz
+  sha256: 6f045d7e69a469337e726e2d2296bea7167bdd4daa1761cdba2e0d1b456d5259
+
+build:
+  script: {{ PYTHON }} -m pip install -vv .
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python      <=3.8.12
+    - pywin32     # [win]
+    - ffmpeg      # [unix]
+    - ipycanvas   <=0.10.2
+    - ipycytoscape <=1.3.2
+    - ipyevents   <=2.0.1
+    - ipytree     <=0.2.1
+    - ipywidgets  <=7.6.5
+    - jupyterlab  >=3.0
+    - numpy       <=1.19.3,>=1.13 # Constrained by paddlepaddle
```

Comment:
My app includes a button which allows the user to download a supplementary library which imposes this version constraint on numpy.
I was advised that it was a good idea to cap all other dependencies to a known working configuration if there are any strict requirements like this.
This numpy constraint definitely needs to stay, but let me know if you think I should remove the others